README.adoc

```diff
@@ -37,12 +37,14 @@ This installer is to be downloaded from https://wiki.eclipse.org/Eclipse_Install
 Run the installer and apply the following steps:
 
 1. change to "Advance Mode" via the menu (upper-right corner) (no need to move the installer)
-2. select a product, e.g. "Eclipse IDE for Eclipse Committers" with product version "Oxygen"
+2. select a product, e.g. "Eclipse IDE for Eclipse Committers" with latest version (2019-06)
```

Comment:
I'd recommend to avoid the term "latest", here, because in the drop down in Oomph provides an option with "latest" in its name that will make the installed Eclipse instance update the moment a new Eclipse version is published, which may break the installation (in the Oomph sense "latest" means, "use latest version at time of installation AND automatically updated to newly released version in the future").

Reply:
I just checked the Oomph UI:
![image](https://user-images.githubusercontent.com/7124687/62516131-7a32e500-b824-11e9-85d9-abed9522488c.png)
and the options of the drop down menu:
![image](https://user-images.githubusercontent.com/7124687/62516155-90d93c00-b824-11e9-96d3-c19958062fe3.png)

Reply:
So I'd change the line
```
2. select a product, e.g. "Eclipse IDE for Eclipse Committers" with product version "Oxygen"
```
to something like
```
2. select a product, e.g. "Eclipse IDE for Eclipse Committers".
3. in the drop down menu "Product Version", select version "2019-06".
```